services/lagoon-solr/builder.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
       if (_.get(options, '_app._config.uid', '1000') !== '1001') options._app.nonRoot.push(options.name);
 
       const solr = {
-        command: options.command,
+        command: `/sbin/tini -- /lagoon/entrypoints.sh ${options.command}`,
```

Comment:
Does the default command in line 12 not need to then strip the entrypoints call to just be `solr-foreground`